### ISSUE TITLE
Syntax Highlight: which WP plugin to render org2blog source tags?

### ISSUE THREAD

**Issue by priyadarshan** [org2blog/org2blog#240]
I am having a hard time having latest WP version display `org2blog` generated source code with syntax highlight.

I wonder if the README could list which WP syntax highlight plugin will work with Org2blog with latest WordPress (currently 4.8.1)? 

What I have tried:

I have:
```
(setq org2blog/wp-use-sourcecode-shortcode t)
```
For example, a source block like:

```
#+ATTR_WP: :syntaxhl light=true
#+BEGIN_SRC lisp :exports code
(defun open-in-largest-window()
  "Open current buffer in largest window"
  (interactive)
  (let ((oldbuf (current-buffer)))
  (select-window (get-largest-window))
  (switch-to-buffer oldbuf))
)
#+END_SRC
```

Is correctly rendered into:

```
[sourcecode language="lisp" title="" light=true]
(defun open-in-largest-window()
  "Open current buffer in largest window"
  (interactive)
  (let ((oldbuf (current-buffer)))
  (select-window (get-largest-window))
  (switch-to-buffer oldbuf))
)
[/sourcecode]
```

But  the final page only show the raw code, as displayed above.

I tried with the following plugins, to no avail:

- WP-Syntax
- WP-GeSHi-Highlight
- WP Code Highlight.js
- Prismatic
- Prism Syntax Highlighter for WordPress
- Enlighter - Customizable Syntax Highlighter

Which syntax highlight plugin do you guys use on WordPress 4.8.1?

**Comment by grettke** [comment]
@priyadarshan I don't use one, but I just configured one and it works fine. I started here https://en.support.wordpress.com/code/posting-source-code/ and ended up here https://wordpress.org/plugins/syntaxhighlighter/#description and that worked. Screenshots of the code and the result are attached.

![screenshot 2017-08-26 19 51 46](https://user-images.githubusercontent.com/2521647/29746211-49f586ce-8a98-11e7-8f08-6815a76a99e7.png)
![screenshot 2017-08-26 19 52 02](https://user-images.githubusercontent.com/2521647/29746210-49f475a4-8a98-11e7-8bff-e605993baf4d.png)

I will update the documentation.
